src/main/java/exceptions/InvalidDateTimeException.java

```diff
@@ -0,0 +1,8 @@
+package exceptions;
+
+public class InvalidDateTimeException extends DukeException {
```

Comment:
Change all Exceptions to `implement` from a single Exception interface (e.g. `IException)

Reply:
But why? Can't DukeException inherit from the main java Exception class? What will the interface do?

Reply:
The idea is because of something called SRP (My original code i just inherited cause it was just for completion of tag). The idea is if I change the implementation of DukeException now but I wish to keep the implementation of others the same, I will run into quite a bit of problems (refactoring). 
We should always follow the idea of whereby the first mistake is fine, just make a new interface. If we didn't make a new interface now, and should we change the implementation of DukeException later on, then its our own fault for not creating an interface earlier.

Reply:
After some research, I realised that its not very easy implementing an Exception. Consider changing everything to extend from `Exception` instead from `DukeException`